server/services/auditService.test.ts

```diff
@@ -0,0 +1,81 @@
+import AuditService from './auditService'
+
+const sqsClientInstance = {
+  send: jest.fn().mockReturnThis(),
+}
+
+jest.mock('@aws-sdk/client-sqs', () => {
+  const { SendMessageCommand } = jest.requireActual('@aws-sdk/client-sqs')
+  return {
+    SQSClient: jest.fn(() => sqsClientInstance),
+    SendMessageCommand,
+  }
+})
+
+describe('Audit service', () => {
+  let auditService: AuditService
+
+  describe('getVisitors', () => {
```

Comment:
is `getVisitors` the right label for this?

Reply:
Whoops 😆